frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsListComponent.tsx

```diff
@@ -45,7 +48,59 @@ const FieldsListComponent: React.FC<InputModuleProps> = ({ farmDetails }) => {
               <p key={fields.Comment}>{fields.Comment}</p>
             </StyledListItem>
           </StyledCommentContainerMobile>
-          {fieldCount > 2 && <StyledDivider />}
+          <StyledListItemGroupContainer>
+            <StyledListItem width="290px">
+              <h2>Lab (Soil Test Methods)</h2>
+              <p key={fields.SoilTest.TestingMethod}>
+                {fields.hasSoilTest ? fields.SoilTest.TestingMethod : 'N/A'}
+              </p>
+            </StyledListItem>
+            <StyledListItem width="200px">
+              <h2>Sampling Month</h2>
+              <p key={fields.SoilTest.sampleDate}>
+                {fields.hasSoilTest ? fields.SoilTest.sampleDate : 'N/A'}
+              </p>
+            </StyledListItem>
+            <StyledListItemGroup>
+              <StyledListItem width="180px">
+                <h2>NO3-N (ppm)</h2>
+                <p key={fields.SoilTest.valNO3H}>
+                  {fields.hasSoilTest ? fields.SoilTest.valNO3H : 'N/A'}
+                </p>
+              </StyledListItem>
+              <StyledListItem width="180px">
+                <h2>P (ppm)</h2>
+                <p key={fields.SoilTest.valP}>{fields.hasSoilTest ? fields.SoilTest.valP : '25'}</p>
```

Comment:
I would change all these magic numbers to a constant so if you need to edit it later it's much easier to do so.
Named variables also help communicate your intent! To me as a reviewer, '25' means nothing to me :)

Reply:
I left it as 25, as I have to talk to the Product Owner still like what would be the High Values if there isn't a Soil Test or Leaf Test, but I'll just name it highValue for now.